File: INTRODUCCION-POO/EJERCICIO-3/JAVA/Coche.java
class Coche {
  private String marca;
  private String modelo;
  private float velocidad;

  public Coche(String marca, String modelo, float velocidad) {
    this.marca = marca;
    this.modelo = modelo;
    this.velocidad = velocidad;
  }

  public void acelerar() {
    this.velocidad += 10;
  }

  public void frenar() {
    this.velocidad -= 5;
  }

  @Override
  public String toString() {
    return "Coche{" + "marca=" + marca + ", modelo=" + modelo + ", velocidad=" + velocidad + '}';
  }
}

File: INTRODUCCION-POO/EJERCICIO-3/PYTHON/Coche.py
class Coche:
    def __init__(self, marca, modelo, velocidad):
        self.marca = marca
        self.modelo = modelo
        self.velocidad = velocidad

    def acelerar(self):
        self.velocidad += 10

    def frenar(self):
        self.velocidad -= 5

    def __str__(self):
        return f"Coche{{ marca={self.marca}, modelo={self.modelo}, velocidad={self.velocidad} }}"

def main():
    c1 = Coche("Toyota", "Corolla", 150)
    c2 = Coche("Mercedes Benz", "CLA", 100)

    c1.acelerar()
    c2.acelerar()

    c1.frenar()
    c2.frenar()

    print(c1)
    print(c2)

if __name__ == "__main__":
    main()
